### PR TITLE
Improve homepage view

### LINKED ISSUE #5: Homepage (categories)
- [x] display all categories
- [x] When clicked, navigate to /categories/:id

### PR DESCRIPTION
**Pull request by BokChoyWarrior**

### CODE REVIEW

File: frontend/src/routes/Home/index.js [hunk 1]
```diff
@@ -27,19 +28,20 @@ export default function Home() {
   };
 
   if (categories.loading) {
-    return <h1>LOADING!!!</h1>;
+    return <h1>Loading...</h1>;
```
Comment (BokChoyWarrior) [inline frontend/src/routes/Home/index.js:31]: lmao

**Review by BokChoyWarrior — COMMENTED** [review]
I did great

**Review by CodeFeeZilla — APPROVED** [review]
Great work :). You've gone above and beyond, thank you

**Review by hpaterson97 — APPROVED** [review]
gg

**Pull request merged**